misc-utils/cal.1.adoc

```diff
@@ -117,6 +117,9 @@ Display week numbers in the calendar (US or ISO-8601). See the *NOTES* section f
 *--color*[=_when_]::
 Colorize the output. The optional argument _when_ can be *auto*, *never* or *always*. If the _when_ argument is omitted, it defaults to *auto*. The colors can be disabled; for the current built-in default see the *--help* output. See also the *COLORS* section.
 
+*-c, --columns*[=_columns_]::
```

Comment:
It seems like the argument is optional (due to [ ] ), but getopt_long() defines it as required.

Reply:
Fixed.